guacamole/src/main/frontend/src/translations/fr.json

```diff
@@ -633,17 +633,20 @@
         "FIELD_HEADER_SERVER_LAYOUT"   : "Agencement clavier:",
         "FIELD_HEADER_SFTP_DIRECTORY"   : "Répertoire d'upload par défaut:",
         "FIELD_HEADER_SFTP_DISABLE_DOWNLOAD" : "Désactiver le téléchargement de fichier:",
+        "FIELD_HEADER_SFTP_DISABLE_UPLOAD"        : "Désactiver l'envoi de fichier:",
         "FIELD_HEADER_SFTP_HOST_KEY"    : "Clé publique de l'hôte (Base64):",
         "FIELD_HEADER_SFTP_HOSTNAME"    : "Nom d'hôte:",
         "FIELD_HEADER_SFTP_SERVER_ALIVE_INTERVAL" : "Intervalle keepalive SFTP:",
         "FIELD_HEADER_SFTP_PASSPHRASE"  : "Phrase secrète:",
         "FIELD_HEADER_SFTP_PASSWORD"    : "Mot de passe:",
         "FIELD_HEADER_SFTP_PORT"        : "Port:",
         "FIELD_HEADER_SFTP_PRIVATE_KEY" : "Clé privée:",
+        "FIELD_HEADER_SFTP_PUBLIC_KEY"  : "Clé publique:",
         "FIELD_HEADER_SFTP_ROOT_DIRECTORY" : "Dossier racine de l'explorateur de fichier:",
-        "FIELD_HEADER_SFTP_DISABLE_UPLOAD"        : "Désactiver l'envoi de fichier:",
+        "FIELD_HEADER_SFTP_TIMEOUT"        : "Délai d'expiration de la connexion SFTP:",
         "FIELD_HEADER_SFTP_USERNAME"    : "Identifiant:",
         "FIELD_HEADER_STATIC_CHANNELS"  : "Noms des canaux statiques:",
+        "FIELD_HEADER_TIMEOUT"         : "Délai d'expiration de la connexion",
```

Comment:
It looks like there's one missing space here.

Reply:
Added!